lndb_storage/object/_subset_anndata.py

```diff
@@ -96,3 +98,40 @@ def _subset_anndata_file(
         adata = _subset_adata_storage(storage, query_obs, query_var)
 
     return adata
+
+
+def get_obs_var_storage(
+    storage: Union[zarr.Group, h5py.File], which=Literal["obs", "var"]
+) -> pd.DataFrame:
+    with storage as access:
+        array = read_elem(access[which])
+    if "index" in array.dtype.fields:
+        return pd.DataFrame.from_records(array, index="index")
```

Comment:
Strange, it should be DataFrame. Probably it is array only for older adatas.

Reply:
It returned an array for my example. Hm. Should I add a test?

Reply:
You're right:
![Uploading image.png…]()

Reply:
Gives me an error here.

Reply:
It seems to be array only for old adatas.